src/Compute/Compute.Test/ScenarioTests/VirtualMachineTests.cs

```diff
@@ -283,5 +283,19 @@ public void TestLowPriorityVirtualMachine()
         {
             TestRunner.RunTestScript("Test-LowPriorityVirtualMachine");
         }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestSetAzVMOperatingSystem()
+        {
+            TestRunner.RunTestScript("Test-SetAzVMOperatingSystem");
+        }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestSetAzVMOperatingSystemError()
+        {
+            TestRunner.RunTestScript("Test-SetAzVMOperatingSystemError");
+        }
```

Comment:
add two tests to check -PatchMode behavior for the cmdlet. One to check default value, the other one to check when it should error out